staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -337,5 +361,10 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"This option, if set, represents the maximum amount of grace period the apiserver will wait "+
 		"for active watch request(s) to drain during the graceful server shutdown window.")
 
-	utilfeature.DefaultMutableFeatureGate.AddFlag(fs)
+	utilversion.DefaultComponentGlobalsRegistry.AddFlags(fs)
```

Comment:
At a glance, why isn't this `s.ComponentGlobalsRegistry`?

Reply:
oops, fixed.